src/views/PianoRoll/SelectionGui.lua

```diff
@@ -0,0 +1,69 @@
+local UID = dofile(views_path .. "PianoRoll/UID.lua")
+local PianoRoll = dofile(views_path .. "PianoRoll/PianoRoll.lua")
+
+return {
+    Render = function()
+        for i = 1, #PianoRollContext.all, 1 do
+            if (ugui.button({
+                uid = UID.SelectPianoRoll + i - 1,
+
+                rectangle = grid_rect(i - 1, 14, 1, 1),
+                is_enabled = true,
+                text = tostring(i)
+            })) then
+                PianoRollContext.current = PianoRollContext.all[i]
+            end
+        end
+        if (#PianoRollContext.all < 8 and ugui.button({
+            uid = UID.AddPianoRoll,
+
+            rectangle = grid_rect(#PianoRollContext.all, 14, 1, 1),
+            is_enabled = true,
+            text = '+'
+        })) then
+            PianoRollContext.current = PianoRoll.new(GetGlobalTimer())
+            PianoRollContext.all[#PianoRollContext.all + 1] = PianoRollContext.current
+        end
+
+        if PianoRollContext.current ~= nil then
+            if (ugui.button({
+                uid = UID.SavePianoRoll,
+
+                rectangle = grid_rect(0, 13.5, 1.5, 0.5),
+                is_enabled = true,
+                text = 'Save'
+            })) then
+                ugui.end_frame()
```

Comment:
Has that fix been publicly released yet, other than as a build artefact on Github?  
Either way, I'd probably want to wait for at least another Repack release before removing this workaround, because most people would still run into the issue.

I'd suggest adding a `--FIXME` comment and opening an issue for this specifically that can be resolved once the fixed mupen build is widely enough available.

Reply:
That's good.